README.md

```diff
@@ -1,3 +1,29 @@
 # hw01
 
-My name is Ahyeon Cho.
\ No newline at end of file
+# Brief biography 
+## Introduction
+My name is Ahyeon Cho (*Ah* is pronounced like the *o* in the word otter, *yeon* is pronounced y + uh + n). I am from South Korea.
+
+## Interesting things to know about me
+
+1. I am a die-hard **Friends** fan
+2. I do not like *steamed* carrots 
+3. But I do like *fresh* carrots
+4. I memorize almost all **90s disney songs** by heart
+5. The follwing image is what my life has been like since quarantine
+
+![Addiction to Zelda](https://pmcvariety.files.wordpress.com/2018/04/lzbotw_tvc_2.jpg)
+
+## Reasons for taking this course
+I am *very* technologically challenged. I learned much later compared to my peers how to capture an image on my phone. The following are the few things that I am able to do using my computer:
+
+- search on Google
+- read and send e-mail
+- use word and powerpoint 
+- watch Youtube videos
+
+However, I realize that in the 21st century, I would be a much more attractive candidate in both the job market and academia if I knew how to use a programming tool. (I do not even know if that is what you call programs like R, Stata, python etc.) So, in an attempt to become a more 21st century type employee, I committed to taking this course for my last quarter in MAPSS.
+
+_____________________________
+
+[Link to Youtube video I watched for reference](https://www.youtube.com/watch?v=6A5EpqqDOdk "Go to youtube link")
```

Comment:
I didn't know that you could make the text that shows up when you hover over a link say something different. This was **very very** cool. Excellent addition!!